acceptance/openstack/autoscaling/v1/lifecycle_hooks_test.go

```diff
@@ -0,0 +1,73 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/clients"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/openstack/autoscaling"
+	"github.com/opentelekomcloud/gophertelekomcloud/acceptance/tools"
+	lifecyclehooks "github.com/opentelekomcloud/gophertelekomcloud/openstack/autoscaling/v1/lifecycle_hooks"
+	th "github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+)
+
+func TestLifecycleHooksLifecycle(t *testing.T) {
+	client, err := clients.NewAutoscalingV1Client()
+	th.AssertNoErr(t, err)
+
+	networkID := clients.EnvOS.GetEnv("NETWORK_ID")
```

Comment:
Please put this check before client init.
If env vars are missing the client init becomes redundant.